lib/md045.js

```diff
@@ -26,5 +28,26 @@ module.exports = {
         );
       }
     }
+
+    const html = filterByTypes(params.parsers.micromark.tokens, [ "htmlText" ]);
+    for (const token of html) {
+      const htmlTagInfo = getHtmlTagInfo(token);
+      if (htmlTagInfo && htmlTagInfo.name.toLowerCase() === "img") {
+        const hasAltAttribute = new RegExp(/alt=/i);
+        if (!hasAltAttribute.test(token.text)) {
```

Comment:
Looks like this will only error if the alt text is missing entirely. Should we also error if the alt text is empty (`alt=""`)? Or is that technically OK since it could indicate a presentational image?

Reply:
I suggest it is reasonable as implemented because it is clear in that case that the author considered what the alternate link should be. However, the existing implementation looks like it will flag empty string which would be inconsistent. However, if I recall correctly (on phone currently) that may be due to how the parser represents this rather than a deliberate decision to flag present-but-empty alternate text.

Reply:
Yes, my intention is for this rule to allow decorative images (set via empty strings).